user-console/src/main/java/org/pentaho/mantle/client/commands/RunInBackgroundCommand.java

```diff
@@ -459,26 +442,55 @@ public void onResponseReceived( Request request, Response response ) {
-          PerspectiveManager.SCHEDULES_PERSPECTIVE ) ) {
-        ScheduleCreateStatusDialog successDialog = new ScheduleCreateStatusDialog();
-        successDialog.center();
-      } else {
-        MessageDialogBox dialogBox =
-            new MessageDialogBox(
-                Messages.getString( "scheduleUpdatedTitle" ), Messages.getString( "scheduleUpdatedMessage" ), //$NON-NLS-1$ //$NON-NLS-2$ 
-                false, false, true );
-        dialogBox.center();
-      }
+  private static native void setupNativeHooks( RunInBackgroundCommand cmd )
+  /*-{
+    $wnd.mantle_runInBackgroundCommand_setOutputName = function(outputName) {
+      //CHECKSTYLE IGNORE LineLength FOR NEXT 1 LINES
+      cmd.@org.pentaho.mantle.client.commands.RunInBackgroundCommand::setOutputName(Ljava/lang/String;)(outputName);
+    }
+
+    $wnd.mantle_runInBackgroundCommand_setOutputLocationPath = function(outputLocationPath) {
+      //CHECKSTYLE IGNORE LineLength FOR NEXT 1 LINES
+      cmd.@org.pentaho.mantle.client.commands.RunInBackgroundCommand::setOutputLocationPath(Ljava/lang/String;)(outputLocationPath);
+    }
+
+    $wnd.mantle_runInBackgroundCommand_setOverwriteFile = function(overwriteFile) {
+      //CHECKSTYLE IGNORE LineLength FOR NEXT 1 LINES
+      cmd.@org.pentaho.mantle.client.commands.RunInBackgroundCommand::setOverwriteFile(Ljava/lang/String;)(overwriteFile);
+    }
+
+    $wnd.mantle_runInBackgroundCommand_setDateFormat = function(dateFormat) {
+      //CHECKSTYLE IGNORE LineLength FOR NEXT 1 LINES
+      cmd.@org.pentaho.mantle.client.commands.RunInBackgroundCommand::setDateFormat(Ljava/lang/String;)(dateFormat);
     }
-  };
+
+    $wnd.mantle_runInBackgroundCommand_performOperation = function(feedback) {
+      //CHECKSTYLE IGNORE LineLength FOR NEXT 1 LINES
+      cmd.@org.pentaho.mantle.client.commands.RunInBackgroundCommand::performOperation(Z)(feedback);
+    }
+  }-*/;
+
+  private native void createScheduleOutputLocationDialog(String solutionPath, Boolean feedback) /*-{
+   $wnd.pho.createScheduleOutputLocationDialog(solutionPath, feedback);
+  }-*/;
+
+  private native void setOkButtonText() /*-{
+   $wnd.pho.setOkButtonText();
+  }-*/;
+
+  private native void centerScheduleOutputLocationDialog() /*-{
+   $wnd.pho.centerScheduleOutputLocationDialog();
+  }-*/;
+
+  private native void createScheduleParamsDialog( String filePath, JSONObject scheduleRequest, Boolean isEmailConfigValid, Boolean isSchedulesPerspectiveActive ) /*-{
+   $wnd.pho.createScheduleParamsDialog( filePath, scheduleRequest, isEmailConfigValid, isSchedulesPerspectiveActive );
+  }-*/;
+
+  private native void createScheduleEmailDialog( String filePath, JSONObject scheduleRequest ) /*-{
+   $wnd.pho.createScheduleEmailDialog( filePath, scheduleRequest );
+  }-*/;
+
+  private native void getScheduleParams( JSONObject scheduleRequest) /*-{
+   $wnd.pho.getScheduleParams( scheduleRequest );
```

Comment:
@e-cuellar  Nick has found an issue with this code. Please review this slack history
https://hitachivantara-eng.slack.com/archives/C01HJ874L8Z/p1703281047403969

Reply:
Implemented Nic's changes @rmansoor